.github/workflows/main.yml

```diff
@@ -11,22 +19,33 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js LTS
       uses: actions/setup-node@v2
       with:
         node-version: lts/*
 
-    # - name: Use NPM v7
-    #   id: npm7
-    #   run: npm install -g npm@^7
-
     - name: Install Node Package Dependencies
       id: install
-      run: npm i
+      run: npm ci
+
+    - name: Setup zowe.json file
+      id: setup
+      run: |
+        jq '.commandGroups=["zos-files","zos-jobs"]' template/zowe.template.json > zowe.json
+        echo "$(jq '.profiles=["zosmf-profile", "base-profile"]' zowe.json)" > zowe.json
+        cat zowe.json
+        npm run contribute
+
+    - name: Get Zowe version
+      id: get-version
+      run: |
+        curl -s https://raw.githubusercontent.com/zowe/zowe-cli-standalone-package/master/zowe-versions.yaml > __zowe__version__
+        echo "::set-output name=number::$(cat __zowe__version__ | grep zowe: | cut -d ':' -f 2)"
+        rm __zowe__version__
 
     - name: Build Source
       id: build
-      run: npm run build --if-present
+      run: npm run build:local -- ${{ github.event.inputs.zowe-version || steps.get-version.outputs.number || '__ERROR__' }}
```

Comment:
Do we want` __ERROR__`, or do we want it to default to some value (i.e. `CLI v` + the output of `zowe --version`)?